src/TileChartMap.jsx

```diff
@@ -42,8 +42,10 @@ export class TileChartMap extends React.Component {
       // chart builder uses fullName
       name: fullName
     }
-    if ( !compProps.selectedState || compProps.selectedState.abbr !== abbr ) {
-      compProps.mapShapeToggled( selectedState )
+    if ( compProps.stateFilters.includes( abbr ) ) {
+      compProps.removeState( selectedState )
+    } else {
+      compProps.addState( selectedState )
```

Comment:
This actually makes more sense and obvious with the visual cues.  I like how this is demonstrated in the  [gif](https://www.adweek.com/creativity/jif-partnered-with-giphy-to-make-a-limited-edition-peanut-butter-no-one-can-pronounce/)